File: apps/connect4-api/src/main/kotlin/com/connect4/model/GlobalStats.kt
package com.connect4.model

data class GlobalStats(
  val averageGameTime: Double,
  val minGameTime: Long,
  val maxGameTime: Long,
  val averageMoves: Double,
  val minMoves: Int,
  val maxMoves: Int,
  val redWins: Int,
  val yellowWins: Int,
  val draws: Int,
)

File: apps/connect4/src/app/features/jobs/models/GlobalStats.ts
export class GlobalStats {
  constructor(
    public averageGameTime: number,
    public minGameTime: number,
    public maxGameTime: number,
    public averageMoves: number,
    public minMoves: number,
    public maxMoves: number,
    public redWins: number,
    public yellowWins: number,
    public draws: number,
  ) {
  }
}
